libcaf_core/src/json_reader.cpp

```diff
@@ -297,13 +299,16 @@ bool json_reader::begin_field(std::string_view name,
       return false;
     }
   } else {
+    // Always push since we'll do a .pop_back() in end_field().
+    field_.push_back(name);
     return false;
   }
 }
 
 bool json_reader::begin_field(std::string_view name, bool& is_present,
                               span<const type_id_t> types, size_t& index) {
   SCOPE(position::object);
+  field_.push_back(name);
```

Comment:
In this function, `push_back(name)` can happen twice. Here and in line 320. Is that correct?

Reply:
Oops. Fixed.